netlify.toml

```diff
@@ -0,0 +1,7 @@
+[[plugins]]
+package = "netlify-plugin-checklinks"
+
+    [plugins.inputs]
+    pretty = true
+
+    skipPatterns = ['open-wc-org.netlify.app']
```

Comment:
Links directly to the new preview build will inherently fail because it's not been published yet, so skip them.